src/qtism/data/storage/xml/marshalling/TemplateControlMarshaller.php

```diff
@@ -49,20 +51,26 @@ class TemplateControlMarshaller extends RecursiveMarshaller
     /**
      * @param DOMElement $element
      * @param QtiComponentCollection $children
+     *
      * @return mixed
-     * @throws MarshallerNotFoundException
      * @throws UnmarshallingException
      * @throws ReflectionException
+     *
+     * @throws MarshallerNotFoundException
      */
-    protected function unmarshallChildrenKnown(DOMElement $element, QtiComponentCollection $children)
+    protected function unmarshallChildrenKnown(DOMElement $element, QtiComponentCollection $children): QtiComponent
     {
         $expressionElts = $this->getChildElementsByTagName($element, Expression::getExpressionClassNames());
 
         if (count($expressionElts) > 0) {
             $marshaller = $this->getMarshallerFactory()->createMarshaller($expressionElts[0]);
             $expression = $marshaller->unmarshall($expressionElts[0]);
-        } elseif (($element->localName == 'templateIf' || $element->localName == 'templateElseIf') && count($expressionElts) == 0) {
-            $msg = "An '" . $element->localName . "' must contain an 'expression' element. None found at line " . $element->getLineNo() . "'.";
+        } elseif (($element->localName == 'templateIf' || $element->localName == 'templateElseIf') && count(
+                $expressionElts
+            ) == 0) {
```

Comment:
could you provide here proper multiline if statement as suggested earlier or revert that place?